prow/config/config_test.go

```diff
@@ -902,7 +902,25 @@ func TestValidatePodSpec(t *testing.T) {
 			spec: func(s *v1.PodSpec) {
 				s.Containers[0].VolumeMounts = append(s.Containers[0].VolumeMounts, v1.VolumeMount{
 					Name:      "foo",
-					MountPath: filepath.Join(decorate.VolumeMountPaths().List()[0], "extra"),
+					MountPath: filepath.Join(decorate.VolumeMountPathsOnTestContainer().List()[0], "extra"),
+				})
+			},
+		},
+		{
+			name: "accept non-conflicting mount path",
+			spec: func(s *v1.PodSpec) {
+				s.Containers[0].VolumeMounts = append(s.Containers[0].VolumeMounts, v1.VolumeMount{
```

Comment:
These two tests test the same thing, did you mean to set the `Volume` here?

Reply:
The meaning was different but yeah it's the same underlying data